src/ServicePulse.Host/app/js/dashboard/eventLogItems.ts

```diff
@@ -0,0 +1,13 @@
+import EventLogItem from "./eventLogItem";
+
+const scu = 'http://localhost:33333/api/';
+
+export default class EventLogItems {
+    getEventLogItems() : Promise<EventLogItem[]> {
+        var url = scu + 'eventlogitems';
+
+        return fetch(url).then<EventLogItem[]>(response => {
```

Comment:
Would you recommend using vanilla fetch or something like axios?

Reply:
I'm not terribly opposed. I haven't used axios, but have used fetch plentifully. So I'm not sure that axios adds anything that we'd be missing for ServicePulse.

A quick search shows

- Easier cancellation (definitely relevant for a SPA, but maybe less so for a MPA)
- Progress bars for large transfers (not relevant for ServicePulse _yet_)
- Support for IE (Not needed for ServicePulse)
- One less step in json transformation (with fetch you have to call `response.json()` to get the json result)

None of these feel like killer features to me. I'm also a little wary since we've had at least [1](https://github.com/Particular/ServicePulse/issues/1184) bug recently that was awkward to fix because we were using non-standard ways of retrieving data. I could be convinced, but my default is to stick with standards until they no longer work.